OSAT-VideoCompositor/Classes/AVPlayerLayer.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  VideoPlayer.swift
+//  OSAT-VideoCompositor
+//
+//  Created by Rohit Sharma on 13/10/22.
+//
+
+import AVFoundation
+import UIKit
+
+public protocol AVPlayerProtocol: AnyObject {
+    func play()
+    func pause()
+    func set(url: URL)
+}
+
+public typealias AVPlayerCustomView = UIView & AVPlayerProtocol
+
+open class AVPlayerView: AVPlayerCustomView {
+    private var player: AVPlayer?
+    private var playerLayer: AVPlayerLayer?
+    private var url: URL?
```

Comment:
please in mind that, right now it's only for single video. we should to make it as multiple videos in next updates

Reply:
For now, we can start with 1 video (later on we can update it)